### PR TITLE
Deploy to Amazee

### PR DESCRIPTION
**Pull request by zendoodles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,39 +3,40 @@
...
+You will be prompted to provide the passphrase for your ssh key. Comply.
+Next, we need to make sure your shell knows about some Docker environment variables:
 
-1. From inside the project root, run:
- * `composer install`
- * `vagrant up`
-1. You will be prompted for the administration password on your host machine. Obey.
-1. SSH in and install the site:
+```
+eval(cachalot env)
```
Comment (froboy) [inline README.md]: Should be `eval $(cachalot env)`

File: README.md [hunk 2]
```diff
@@ -3,39 +3,40 @@
...
-* [vagrant-auto_network](https://github.com/oscar-stack/vagrant-auto_network) `vagrant plugin install vagrant-auto_network`
+###  Development Environment
 
-If you have been running a previous version of Vagrant you may need to do: `vagrant plugin update` to ensure that you can install the plugins.
+This site uses the Amazee.io Drupal Docker development environment. If you already have this, you can skip this bit entirely. Their [documentation for setup](https://docs.amazee.io/local_docker_development/local_docker_development.html) is amazeballz, but if you just want to [quick get going on your Mac](https://stories.amazee.io/easy-local-drupal-development-on-os-x-a01a343f99e3):
 
-## Getting Started
+```
+brew tap amazeeio/cachalot
+brew install docker docker-machine cachalot
```
Comment (froboy) [inline README.md]: Need `docker-compose` in here too.

File: README.md [hunk 3]
```diff
@@ -3,39 +3,40 @@
...
+cachalot create --provider virtualbox
```
Comment (labbydev) [inline README.md]: `Valid values are: 'virtualbox', 'vmware', 'xhyve', or 'parallels'`

**Comment by froboy** [comment]
I've added a patch from https://www.drupal.org/node/2907654 that blocks config_installer from successfully importing config and also added `admin-user` to the site install so that user 1 has the expected username.

**Pull request merged**